dot/sync/disjoint_block_set.go

```diff
@@ -23,6 +23,8 @@ var (
 	errSetAtLimit   = errors.New("cannot add block; set is at capacity")
 )
 
+//go:generate mockgen -destination=mock_disjoint_block_set.go -package=$GOPACKAGE . DisjointBlockSet
```

Comment:
Write the mock to a `_test.go` file:

```suggestion
//go:generate mockgen -destination=mock_disjoint_block_set_test.go -package=$GOPACKAGE . DisjointBlockSet
```

Reply:
Updated.